axelrod/fingerprint.py

```diff
@@ -0,0 +1,206 @@
+import axelrod as axl
+import numpy as np
+import pandas as pd
+import seaborn as sns
+import matplotlib.pyplot as plt
+from itertools import product
+from axelrod.strategy_transformers import JossAnnTransformer, DualTransformer
+from axelrod.interaction_utils import compute_final_score_per_turn as cfspt
+from collections import OrderedDict
+
+
+class Fingerprint():
+    def __init__(self):
+        pass
+
+    def plot(self):
+        pass
+
+
+class AshlockFingerprint(Fingerprint):
+    def __init__(self, strategy, probe):
+        """
+        Parameters
+        ----------
+        strategy : class
+            A class that must be descended from axelrod.strategies.
+        probe : class
+            A class that must be descended from axelrod.strategies
+        """
+        self.strategy = strategy
+        self.probe = probe
+
+    def create_probe_coords(self, granularity):
```

Comment:
`granularity` should be `steps`.

An increase of granularity here would imply a less precise image/fingerprint but that's not usually what is meant by granularity.